### PR TITLE
fix: ContainerAware for Symfony 2.8/3

### PR DESCRIPTION
**Pull request by gsdevme**

### CODE REVIEW

**Comment by shieldo** [comment]
Should we drop Symfony 2.7 support? It's still in LTS. We don't need to use the trait to implement `ContainerAwareInterface`.

File: composer.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.1",
-        "symfony/finder": "~2.1",
-        "symfony/yaml": "~2.3",
-        "symfony/console": "~2.3",
-        "symfony/process": "~2.3",
+        "symfony/framework-bundle": "~2.8||~3.0",
```
Comment (shieldo) [inline composer.json]: Have you tested this syntax? Pretty sure you would need a single `|`

Comment (gsdevme) [inline composer.json]: I've ran a composer on it, but perhaps a single is correct? I guess they might both work in theory.

**Comment by gsdevme** [comment]
Im not saying this should be merged right this second. I don't see a point in supporting 2.7 personally though, we could provide two versions though.

**Comment by shieldo** [comment]
I expect there are other people using this bundle, and therefore we should support LTS versions.  At least we can make 1.1 (for 2.7) and 1.2/master (for 2.8) branches.

**Comment by gsdevme** [comment]
Did we not just make a pretty big change to this bundle however? I feel like those people would be using 1.x still, without further documentation changing to 2.x will be tricky.

**Comment by shieldo** [comment]
Closing as class no longer extends `ContainerAware`, and bundle is not compatible with 3.0 yet (mainly due to non-compliant form types).

**Pull request closed without merging**